README.md

```diff
@@ -112,7 +112,7 @@ return [
                 'host' => env('DB_HOST_READONLY'),
             ] : null,
             'write' => [],
-            'sticky' => true,
+            'sticky' => (bool)env('DB_HOST_READONLY'),
```

Comment:
Should be a problem if the user writes a value different from true or false?

Reply:
It's okay. Dotenv parser and `(bool)` casting always forces `true` or `false`.

## Expected values

- `"xxx.ap-northeast-1.rds.amazonaws.com"` -> true
- `""` -> false
- `"null"` -> false
- Missing -> false

## Malformed values (Improper for this)

- `"true"` -> true
- `"false"` -> false
- `"tru"` -> true
- `"fal"` -> true
- `"1"` -> true
- `"0"` -> false
- `"-1"` -> false